2-mandatory/1-create-functions.js

```diff
@@ -44,7 +50,18 @@ Write a function that:
 - Numbers greater 100 must be replaced with 100.
 */
 
-function formatPercentage() {
+function formatPercentage(numbers) {
+numbers = numbers.map(num =>{
+  if (num >=100 ) {
+    num = 100; + "%";
+  }
+  else if ( num % 1 == 0){
+     num = num + "%";
+  } else{
+    num = num.toFixed(2) + "%";
+  }
+});
+
```

Comment:
Your function needs to return the `numbers` and also you need extra if state to cover numbers that looks like this `18.10` which Fatih share he solution in the Slack channel!

Thanks